odoo/fields.py

```diff
@@ -2924,7 +2962,7 @@ def convert_to_write(self, value, record):
                         for name in record._cache
                         if name not in inv_names
                     })
-                    result.append((0, 0, values))
+                    result.append((X2ManyCmd.CREATE, 0, values))
```

Comment:
```suggestion
                    result.append(Command.create(values))
```
and so on

Reply:
I agree, this is cleaner... but also more complicated for my script to rewrite :p